crypto/bloom/timestamp_bloom.go

```diff
@@ -0,0 +1,145 @@
+package bloom
```

Comment:
maybe name the file `timestamp.go` because it's already in package `bloom`?

Reply:
yeah, make sense